__tests__/helpers.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const MemoryFileSystem = require("memory-fs");
+const webpack = require("webpack");
+
+exports.PluginEnvironment = class PluginEnvironment {
```

Comment:
How does `result.output` look like in (terms of noise), could you post it here when reddy ? 😛 
If we agree on 'extending' a base config we could evetually add `CommonChunksPlugin`to the base to split the webpack runtime by default

```js
[ 
  new CommonChunksPlugin({ 
    name: [ 'runtime', `${fixture}` ], 
    minChunks: Infinity 
  }) 
].concat(config.plugins || [])
```

```js
webpackJsonp([`${fixture}`],{...bundle}
```